nsxt/resource_nsxt_ip_pool_allocation_ip_address_test.go

```diff
@@ -0,0 +1,140 @@
+/* Copyright © 2018 VMware, Inc. All Rights Reserved.
+   SPDX-License-Identifier: MPL-2.0 */
+
+package nsxt
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"github.com/vmware/go-vmware-nsxt"
+	"math/rand"
+	"net/http"
+	"testing"
+	"time"
+)
+
+const (
+	// how often to check destroyment of allocated ip address
+	destroyTestPercent = 10
+
+	// how long to wait for updated allocation IP address list (to check destroyment)
+	waitSeconds = 120
+)
+
+func TestAccResourceNsxtIpPoolAllocationIPAddress_basic(t *testing.T) {
```

Comment:
go linter requires capital letters for "IP", please run a linter check to find all cases